admin/load.php

```diff
@@ -403,6 +403,41 @@ function perflab_admin_pointer( $hook_suffix ) {
 	$current_user = get_current_user_id();
 	$dismissed    = explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) );
 
+	/*
+	 * Show an admin pointer if module is active to prompt
+	 * for an action to use the standalone plugin instead.
+	 */
+	$available_module_names = perflab_get_modules_compatible_with_standalone_plugin();
+	if (
+		! empty( $available_module_names )
+		&& current_user_can( 'install_plugins' )
+		&& current_user_can( 'activate_plugins' )
+		&& ! in_array( 'perflab-module-migration-pointer', $dismissed, true )
+	) {
+		// Enqueue the pointer logic and return early.
+		wp_enqueue_style( 'wp-pointer' );
+		wp_enqueue_script( 'wp-pointer' );
+		add_action(
+			'admin_print_footer_scripts',
+			static function () {
+				$content  = __( 'Your site is using modules which will be removed in the future in favor of their equivalent standalone plugins.', 'performance-lab' );
+				$content .= ' ' . sprintf(
+					/* translators: %s: settings page link */
+					__( 'Open %s to learn more about next steps to keep the functionality available.', 'performance-lab' ),
+					'<a href="' . esc_url( add_query_arg( 'page', PERFLAB_MODULES_SCREEN, admin_url( 'options-general.php' ) ) ) . '">' . __( 'Settings > Performance', 'performance-lab' ) . '</a>'
+				);
+				perflab_render_pointer(
+					'perflab-module-migration-pointer',
+					array(
+						'heading' => __( 'Action required', 'performance-lab' ),
```

Comment:
Maybe use `Performance Lab` (or `Performance Lab: Action required` or so) as the heading? Makes it more clear which plugin the pointer is coming from.

Reply:
Thank you for the swift review.

I've made an update to the pointer heading. While we implemented a similar pointer previously in #739 for the SQLite module and omitted it, considering the overall migration, it seems sensible to include it now.